### ISSUE TITLE
I reached the cluster limit on coiled

### ISSUE THREAD

**Issue by cspencerjones** [ocean-transport/coiled_collaboration#14]
My coiled dashboard now says "As you have reached your quota of Coiled credits, any running clusters will be stopped shortly. Your credit quota renews on June 13, 2021." And I can't start new clusters. I have been burning quite a lot of time in order to diagnose what's not working for me (I now think I have a combination of issues: too many tasks in general, plus some slightly heterogeneous data). 

@jrbourbeau is there any possibility we could raise my limit? (I should note that @rabernat is on holiday today and I haven't talked to him about this).

**Comment by jrbourbeau** [comment]
Just increased your limit! Let me know if you get need another bump later on

> @rabernat is on holiday today

🎉

**Comment by jbusecke** [comment]
Should I ask for this preemptively? Or when the time comes (probably next week).

**Comment by jrbourbeau** [comment]
Just preemptively bumped your limit @jbusecke

**Comment by jbusecke** [comment]
Thx so much @jrbourbeau. Enjoy your weekend yall!

**Comment by jrbourbeau** [comment]
Yeah, no worries -- I suspect you'll be able to put those AWS nodes to good use : )

**Comment by mrocklin** [comment]
Out of curiosity, are these on our account, or are we deploying into a LDEO AWS account?  I'm fine either way, but if y'all have academic credits to burn I'd rather we did that :)

**Comment by jrbourbeau** [comment]
I believe they are on our account, though it'd be good for @jbusecke or @cspencerjones to confirm. What do your "Cloud Backend Options" settings look like for you "Account" page? Specifically, does the "Credentials" entry say "Default (use Coiled's)"?

<img width="1552" alt="Screen Shot 2021-06-10 at 5 52 28 PM" src="https://user-images.githubusercontent.com/11656932/121606955-bf259500-ca14-11eb-889f-05e7c1281e58.png">

**Comment by cspencerjones** [comment]
Yes, it says "Default (use Coiled's)"

**Comment by jbusecke** [comment]
Same here
